### ISSUE TITLE
Part 4: Support for JSON Patch (RFC 6902)

### ISSUE THREAD

**Issue by rhysrevans3** [opengeospatial/ogcapi-features#957]
The current specification suggests PATCH endpoints should use [JSON Merge Patch](https://portal.ogc.org/files/108198#_json_merge_patch) ([RFC 7396](https://datatracker.ietf.org/doc/html/rfc7396)).

However in JSON Merge Patch `null` values are given special meaning to indicate the removal of existing values in the target.

Which isn't recommended for documents that allow explicit `null` values which I believe OGC Features allow.

```
This design means that merge patch documents are suitable for
describing modifications to JSON documents that primarily use objects
for their structure and do not make use of explicit null values.
```

Could JSON Patch ([RFC 6902](https://datatracker.ietf.org/doc/html/rfc6902)) be supported alongside JSON Merge Patch (RFC 7396)? Which allows for operations (add, remove, replace, move, copy, test) to be declared which allows the setting of null values.

The `Accept/Content-Type` header for JSON Patch (`application/json-patch+json`) is different than that of JSON Merge Patch (`application/merge-patch+json`) so these can both be supported at the same time.

Alternately, can it be made explicit that although JSON Merge Patch is recommended it does not have to be used exclusively to the exclusion of any other standards.

**Comment by pvretano** [comment]
@rhysrevans3, yes, JSON Patch could be used alongside JSON Merge Patch.  I would say you should advertise the appropriate JSON Patch media type at the appropriate endpoint in the server's OpenAPI document but beyond that you should be good to go.

**Comment by pvretano** [comment]
We should still probably add a disclaimer that JSON Merge Patch is recommended but not exclusive.  My server, for example, also supports the old WFS Transaction encoding for updates alongside JSON Merge Patch.
